### PR TITLE
document oracle pki provider

### PR DESCRIPTION
**Pull request by CarolynRountree**
Reference Oracle PKI Provider document

### CODE REVIEW

File: documentation/2.0/content/rcuinfo.md [hunk 1]
```diff
@@ -63,6 +63,9 @@ domainInfo:
       oracle.net.tns_admin: <absolute path of the unzipped wallet root directory>
 
 
+When using PKCS12 wallet, you must include the Oracle PKI provider to access your wallet. Add the Oracle PKI Provider to your JAVA java.security file. Refer to `section 2.2.4` of the following document for information about adding the Oracle PKI Provider to the JAVA java.security file.
```
Comment (ddsharpe) [inline documentation/2.0/content/rcuinfo.md:66]: I think this paragraph should be outside the code block.

**Review by ddsharpe — CHANGES_REQUESTED** [review]

**Review by rosemarymarano — COMMENTED** [review]
Few edits/suggestions

**Review by rakillen — CHANGES_REQUESTED** [review]

**Review by rakillen — APPROVED** [review]

**Review by ddsharpe — APPROVED** [review]

File: documentation/2.0/content/rcuinfo.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply documentation/2.0/content/rcuinfo.md]: Yes, I missed that.

File: documentation/2.0/content/rcuinfo.md [hunk 2]
```diff
@@ -62,8 +62,9 @@ domainInfo:
       javax.net.ssl.trustStorePassword: <password of the truststore>
       oracle.net.tns_admin: <absolute path of the unzipped wallet root directory>
 
-
 ```
+When using PKCS12 wallet, you must include the Oracle PKI provider to access your wallet. Add the Oracle PKI provider to your Java `java.security` file. For more information about adding the Oracle PKI provider to the Java `java.security` file, see Section 2.2.4 "How can Oracle wallets be used in Java" [SSL with Oracle JDBC](https://www.oracle.com/technetwork/topics/wp-oracle-jdbc-thin-ssl-130128.pdf).
```
Comment (rosemarymarano) [inline documentation/2.0/content/rcuinfo.md:66]: Oops, I missed a small article: "How can Oracle wallets be used in Java" [SSL with Oracle JDBC] -> "How can Oracle wallets be used in Java" **in** [SSL with Oracle JDBC](https://www.oracle.com/technetwork/topics/wp-oracle-jdbc-thin-ssl-130128.pdf).

**Review by rosemarymarano — APPROVED** [review]
One minor fix; otherwise fine.

**Pull request merged**